mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -130,7 +133,7 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	METHOD method_1506 handleGlErrorByDisableVsync (IJ)V
 		ARG 1 error
 		ARG 2 description
-	METHOD method_1507 openScreen (Lnet/minecraft/class_437;)V
+	METHOD method_1507 setScreen (Lnet/minecraft/class_437;)V
```

Comment:
Not sure about this one. It makes it sound as if it was a simple setter. I can understand that openScreen is bad because a null parameter just closes the current one. But given the significance of this one, I'd actually just leave it at openScreen (or maybe showScreen).

Reply:
I'm not sure of the change now. Didn't think at all it could be mislead with a simple setter, which is a fair point. Maybe we should keep openScreen actually, with docs explaining it, even if it's not accurate when you pass a null parameter

Reply:
Yeah maybe keep this rename in a separate pr